plugins/module_utils/_mount.py

```diff
@@ -0,0 +1,90 @@
+# This code is part of Ansible, but is an independent component.
```

Comment:
How do we ensure that this file is kept in sync with `ansible.posix`? to prevent `plugins/modules/system/aix_filesystem.py` being exposed to a bug.

Reply:
A very good question. Maybe we could add an extra sanity test which compares whether the files are equal. Then we'll quickly find out when they update it. (Disadvantage: it will fail for every PR. But then it'll surface more quickly...)

Reply:
@gundalow @Andersson007 as a short-term solution, how about creating an issue for this, and maybe have a column in a project where issues tracking similar problems are tracked? (We could also use a label, but that doesn't work well cross-repository.)

Reply:
sounds good to me